src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -447,18 +450,25 @@ const HoloFuelDnaInterface = {
               type: TYPE.offer,
               actioned: false
             }
-          } else if (JSON.parse(acceptedPaymentHash.Err.Internal).kind.Timeout) {
-            return {
-              ...transaction,
-              id: transactionId, // should always match `Object.entries(result)[0][0]`
-              direction: DIRECTION.incoming, // this indicates the hf recipient
-              status: STATUS.pending,
-              type: TYPE.offer,
-              actioned: true
+          } else {
+            try {
+              const parseableResponse = JSON.parse(acceptedPaymentHash.Err.Internal)
```

Comment:
It will fail here itself so. we do not need to check if `acceptedPaymentHash.Err.Internal` can be parsed.

Just the try catch will do the trick

Reply:
I did this to make it more legible, but I'm fine with having it just check/fail on the if statement instead. 👍